src/routes/HomeScreen.tsx

```diff
@@ -51,19 +44,26 @@ const AddCustomDisciplineText = styled.Text`
   font-family: ${props => props.theme.fonts.contentFontBold};
 `
 
+const Description = styled.Text<{ selected: boolean }>`
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  color: ${props => (props.selected ? props.theme.colors.white : props.theme.colors.lunesGreyMedium)};
+`
+
 interface HomeScreenPropsType {
   navigation: StackNavigationProp<RoutesParamsType, 'Home'>
   customDisciplines: string[]
 }
 
 const HomeScreen = ({ navigation, customDisciplines }: HomeScreenPropsType): JSX.Element => {
-  const [selectedId, setSelectedId] = useState<number | null>(null)
+  const [selectedId, setSelectedId] = useState<string | null>(null)
```

Comment:
Why do we need the id here at all? This screen is not rendered when the user selects something, is it?

Reply:
The selectedId is needed, for highlighting the element you clicked on. Because we now have on the homeScreen elements from customDisciplines and normal Lunes-Disciplines, we need the string because they can have the same ID from the backend. Therefore in the CustomDisciplineMenuItem there is a function mapping the ID of a customDiscipline from `id` to `custom-${id}`, and therefore we need a string here.
Otherwise CustomElement and LunesElement with the same ID will both be highlighted on a click of one of them, which is not what we wanted.

Reply:
Okay not sure if I understand why multiple disciplines can have the same id but ok. But doesn't it make sense then to set the id to `custom-${id}` directly when loading the disciplines from the api?

Reply:
Then we would have to change it back, if we want to make a new request.
E.g. 
Getting a customDiscipline (parent / HomeScreen) -> gets the id = `custom-${id}`
Clicking on this item -> making a request, which needs to be with only `id` and not `custom-${id}`